packages/ui/src/layout/wui-overlay/index.ts

```diff
@@ -7,7 +7,31 @@ import styles from './styles.js'
 export class WuiOverlay extends LitElement {
   public static override styles = [resetStyles, styles]
 
+  public constructor() {
+    super()
+    window.addEventListener('resize', () => this.handleResize())
```

Comment:
Instead of this heavy listener, let's just use media query for height with a pre-set default for max height.  We can take approximate height of "view all" screen which is and will be the tallest and use that.

Additionally some padding at top and bottom will be good, so modal doesn't stick completely to the edges of the screen.

It's ok if at some point modal will need to "jump" to top during flex settings transition in favor of avoiding js listeners.